### PR TITLE
removing the firewall_driver, not a valid parameter

### PR DESCRIPTION
**Pull request by radez**

### CODE REVIEW

File: puppet/modules/quickstack/manifests/neutron/controller.pp [hunk 1]
```diff
@@ -326,10 +325,8 @@
       vxlan_group           => $ml2_vxlan_group,
       vni_ranges            => $ml2_vni_ranges,
       enable_security_group => str2bool_i("$ml2_security_group"),
-      firewall_driver       => $ml2_firewall_driver,
-    }
 
-    # If cisco nexus is part of ml2 mechanism drivers,
+    # If cesco nexus is part of ml2 mechanism drivers,
```
Comment (jguiditta) [inline puppet/modules/quickstack/manifests/neutron/controller.pp]: Bump, if you have a chance to fix this, I'd like to merge, would rather not mess with your commit, or add another just to fix a typo

Comment (radez) [inline puppet/modules/quickstack/manifests/neutron/controller.pp]: Sorry, missed this comment, I'll clean it up. thx for the bump

**Comment by radez** [comment]
fixed the typo

**Comment by jguiditta** [comment]
Great, thanks!

**Pull request merged**